apps/E2E/src/Avatar/specs/Avatar.spec.win.ts

```diff
@@ -16,6 +16,9 @@ describe('Avatar Testing Initialization', function () {
     await AvatarPageObject.waitForPageDisplayed(PAGE_TIMEOUT);
 
     await expect(await AvatarPageObject.isPageLoaded()).toBeTruthy(AvatarPageObject.ERRORMESSAGE_PAGELOAD);
+
+    await AvatarPageObject.enableE2ETesterMode();
```

Comment:
I think we can make some API improvements here. 

We know that every spec is going to need these 3 functions - `NavigateAppPage.clickAndGoToAvatarPage();`, `AvatarPageObject.waitForPageDisplayed(PAGE_TIMEOUT);`, and `await AvatarPageObject.enableE2ETesterMode();`.

Can we consolidate these into one method?

We can just have `NavigateAppPage.navigateToAvatarE2EPage();`. This method is responsible for clicking on the Avatar page, waiting for page load, and enabling E2E tester mode.

I think this makes writing specs a lot cleaner to write and easier to read.

Reply:
Added a method to BasePage: `navigateToPageAndLoadTests()` which handles navigation, waits for the page to load, and if passed a flag, enables e2e mode. 

So far, I only refactored the win32 tests that have e2e sections. Should I do the same for all specs or put that in a separate PR?